vendor/github.com/pganalyze/pg_query_go/v6/parser/build_cgo.go

```diff
@@ -0,0 +1,54 @@
+//go:build required
```

Comment:
See https://github.com/pganalyze/pg_query_go/pull/131 for context on these files (I'll look into dropping modvendor separately in another PR after this one is in)